scripts/npn.py

```diff
@@ -21,54 +27,132 @@ def __init__(self, **kwargs):
         self.name = "USA National Phenology Network"
         self.shortname = "NPN"
         self.retriever_minimum_version = '2.0.dev'
-        self.version = '1.0'
+        self.version = '1.1'
```

Comment:
Actually, since this is such a major change I think this is the right situation to increment the major version number, so use `2.0.0` here instead.